src/main/java/it/pagopa/pn/deliverypush/legalfacts/LegalFactGenerator.java

```diff
@@ -180,11 +180,19 @@ public byte[] generateNotificationViewedLegalFact(String iun, NotificationRecipi
         templateModel.put(FIELD_DISCLAIMER, this.getlegalFactDisclaimer());
 
         return documentComposition.executePdfTemplate(
-                DocumentComposition.TemplateType.NOTIFICATION_VIEWED,
+                retrieveTemplateFromLang(DocumentComposition.TemplateType.NOTIFICATION_VIEWED, notification.getAdditionalLanguages()),
                 templateModel
         );
     }
 
+    private DocumentComposition.TemplateType retrieveTemplateFromLang(DocumentComposition.TemplateType templateType, List<String> lang) {
+        if(CollectionUtils.isEmpty(lang)){
+            return templateType;
+        }
+        String finalTemplateName = templateType.name() + "_" + lang.get(0);
+        return DocumentComposition.TemplateType.valueOf(finalTemplateName);
```

Comment:
Questa logica qui potrebbe aver senso metterla in DocumentCompostion? un metodo che prende in input il template e la lingua (non la lista). In questo modo demandiamo a DocumentComposition totalmente la gestione dei template e della lingua

Inoltre quando a tale metodo viene passato lang.get(0) inserire un commento che specifica che ad oggi viene utilizzato l'additionalLanguages.get(o) perchè è possibile avere una sola lingua aggiuntiva.
Nota se lang.get(0) viene utilizzato in diversi punti, vale la pena inserire il metodo che recupera la lingua a partire dall'array in un utils (per quanto banale) in questo modo abbiamo un unico punto da "modificare" nel momento in cui avremo più lingue da gestire